push_secrets.sh

```diff
@@ -21,19 +21,19 @@ fi
 
 # Fetch all existing secret keys and delete them
 echo "Fetching and deleting all existing secrets..."
-SECRET_KEYS=$(vlt secrets list -format=json | grep -Eo '"([^"]*)"\s*:\s*"([^"]*)"' | sed -E 's/^"([^"]*)"\s*:\s*"([^"]*)"$/\1=\2/' | grep "^name=" | grep -v "@" | sed 's/^name=//')
+SECRET_KEYS=$(hcp vault-secrets secrets list --format=json --app=sistema | grep -Eo '"([^"]*)"\s*:\s*"([^"]*)"' | sed -E 's/^"([^"]*)"\s*:\s*"([^"]*)"$/\1=\2/' | grep "^name=" | grep -v "@" | sed 's/^name=//')
```

Comment:
This works for me, can you check if it works for you?
```suggestion
SECRET_KEYS=$(hcp vault-secrets secrets list --format=json --app=sistema | grep -Eo '"([^"]*)"\s*:\s*"([^"]*)"' | sed -E 's/^"([^"]*)"\s*:\s*"([^"]*)"$/\1=\2/' | grep '^\(name=\|"name":\)'| grep -v "@" | sed 's/^"name": "\(.*\)"$/\1/; s/^name=\(.*\)$/\1/')
```

Reply:
Definitely very weird! My secrets are indeed `name=SECRET_NAME`, which seems to have been the previous default since I didn't make any changes to the `grep` when I went from `vlt` to `hcp`. This change works for me too!

By the way, when I run `hcp version` I get `hcp v0.5.0 (410604f643303ece1593db5d7601a3c4f3a9dd40) go1.22.2 amd64` - could that be different from yours?